### PR TITLE
added UA localization

### PR DESCRIPTION
**Pull request by squidsdev**

### CODE REVIEW

File: README.ua.md [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+  </p>
+
+![Hydra Catalogue](./docs/screenshot.png)
+
+</div>
+
+<br>
+
+### Мова
+[![ua](https://img.shields.io/badge/lang-ua-red)](https://github.com/hydralauncher/hydra/blob/main/README.ru.md)
```
Comment (zamitto) [inline README.ua.md:30]: This badge is for UA but is linking to RU README. 
Could you fix and add the badges for RU and EN readme as weelll?

Comment (squidsdev) [reply README.ua.md]: I fixed the badge for UA. Just a second, I will add RU and EN badges.

**Comment by squidsdev** [comment]
Done!

**Comment by piradata** [comment]
i think what you made here will conflict a little to what i have done in #296 😅

**Comment by piradata** [comment]
@squidsdev , already added everything on the last version of this PR on mine 😄

**Review by zamitto — CHANGES_REQUESTED** [review]
I think you accidently deleted the existing README files. Could you revert these changes, so th only modified files are your translation.json and de src/locales/index.ts file?

**Comment by piradata** [comment]
> I think you accidently deleted the existing README files. Could you revert these changes, so th only modified files are your translation.json and de src/locales/index.ts file? Also, please fix the conflict in the index.ts file

already added then on the PR #296

**Comment by squidsdev** [comment]
Thanks, @piradata

**Comment by piradata** [comment]
by the way @squidsdev, the ukranian loc code is `uk-UA` or `uk`?

**Comment by squidsdev** [comment]
Just checked with the IETF (BCP 47), the correct spelling is `uk-UA`

**Comment by squidsdev** [comment]
Actually, it's hard to say the correct answer to this question, because there are several country codes that are used to designate countries/languages/keyboard layouts.

**Comment by piradata** [comment]
i also found that `uk-UA` seems to be the "most correct" kk, will change here to it

**Pull request closed without merging**